pkgs/dartpad_ui/web/index.html

```diff
@@ -80,26 +75,11 @@
       width: 100%;
     }
   </style>
-
-  <!-- This script adds the flutter initialization JS code -->
-  <script src="flutter.js" defer></script>
 </head>
 
 <body>
   <script>
-    window.addEventListener('load', function (ev) {
-      // Download main.dart.js
-      _flutter.loader.loadEntrypoint({
-        serviceWorker: {
-          serviceWorkerVersion: serviceWorkerVersion,
-        },
-        onEntrypointLoaded: function (engineInitializer) {
-          engineInitializer.initializeEngine().then(function (appRunner) {
-            appRunner.runApp();
-          });
-        }
-      });
-    });
+    {{flutter_bootstrap_js}}
```

Comment:
Er maybe put this here.

`<script src="flutter_bootstrap.js" async></script>`

per https://docs.flutter.dev/platform-integration/web/initialization#flutter_bootstrap-js